src/canvas/canvas.interface.ts

```diff
@@ -68,12 +70,16 @@ export class CanvasMetadataUpdateDTO {
   @MinLength(3)
   @MaxLength(255)
   name: string;
+  @MaxLength(4000)
+  description: string;
```

Comment:
description should be optional field, currently we will get http 400 on create and update endpoints

```json
{
    "message": [
        "description must be shorter than or equal to 4000 characters"
    ],
    "error": "Bad Request",
    "statusCode": 400
}
```

Reply:
@coredumped7893 Done